mandatory/2-function-creation.js

```diff
@@ -45,7 +84,13 @@ function canRegister(age) {}
   )
 */
 
-function countReverse(number) {}
+function countReverse(number) {
+  const minNum = 1;
```

Comment:
Suggestion: create a new variable that you can loop over and decrement rather than using the input `number`. e.g.
```
let currentNumber = number;
while (currentNumber  >= minNum) {
  console.log(number);
  currentNumber--;
}
```
The reason for this is you still have access to the unmodified user input in case you need to use it in  a future modification of the function.
Also you'll find in the future lessons that for certain types passed to a function modifying the input data could have an impact on other parts of the program (this isn't a problem for integers though so wouldn't be an issue here)